scripting/srccoop.sp

```diff
@@ -656,7 +696,7 @@ void GreetPlayer(int client)
 {
 	if (g_pCoopManager.IsFeatureEnabled(FT_SHOW_WELCOME_MESSAGE))
 	{
-		Msg(client, "This server runs SourceCoop version %s.\nPress %s=%s or type %s/coopmenu%s for extra settings.", PLUGIN_VERSION, CHAT_COLOR_SEC, CHAT_COLOR_PRI, CHAT_COLOR_SEC, CHAT_COLOR_PRI);
+		Msg(client, "This server runs SourceCoop version %s.\nPress %s=%s or type %s/coopmenu%s for extra settings. Use !stuck if you get stuck.", PLUGIN_VERSION, CHAT_COLOR_SEC, CHAT_COLOR_PRI, CHAT_COLOR_SEC, CHAT_COLOR_PRI);
```

Comment:
I did wind up using !stuck on a few levels later in Xen, the areas where you have to break a blood cyst to disable some force fields, they bleed and shoot you up using a `func_movelinear` the issue is that this `func_movelinear` also encompasses some of the tunnels around it which can get stuck if someone is in the tunnel when it starts to move.
I can't remember the exact map name at the moment where it is really a problem, I'll check a bit later on.

There are also a couple occurrences during the gonarch chases where the crystals collapse and create a wall, if a player is too far ahead of the others, it can trap the ones behind either behind the wall, or inside the wall.

These are issues that could be fixed in the EDT though, not entirely necessary to have !stuck for those.